tests/Feature/Api/V1/LeaderboardSubmissionsTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use App\Models\Game;
+use App\Models\Leaderboard;
+use App\Models\LeaderboardEntry;
+use App\Models\System;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class LeaderboardSubmissionsTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+
+    public function testItValidates(): void
+    {
+        $this->get($this->apiUrl('GetLeaderboardSubmissions'))
+            ->assertJsonValidationErrors([
+                'i',
+            ]);
+    }
+
+    public function testGetLeaderboardSubmissionsLeaderboardWithNoSubmissions(): void
+    {
+        $this->get($this->apiUrl('GetLeaderboardSubmissions', ['i' => 99999]))
+            ->assertNotFound()
+            ->assertJson([]);
+    }
+
+    public function testGetLeaderboardSubmissions(): void
+    {
+        Carbon::setTestNow(Carbon::now());
+
+        /** Set up a leaderboard with 5 submissions: */
+
+        /** @var System $system */
+        $system = System::factory()->create();
+
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->ID]);
+
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create([
+            'GameID' => $game->ID,
+            'Title' => "Test leaderboard 1",
+            'Description' => "I am the first leaderboard",
+        ]);
+
+        $userOne = User::factory()->create(['User' => 'myUser1']);
+        $leaderboardEntryOne = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userOne->ID,
+        ]);
+
+        $userTwo = User::factory()->create(['User' => 'myUser2']);
+        $leaderboardEntryTwo = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userTwo->ID,
+        ]);
+
+        $userThree = User::factory()->create(['User' => 'myUser3']);
+        $leaderboardEntryThree = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userThree->ID,
+        ]);
+
+        $userFour = User::factory()->create(['User' => 'myUser4']);
+        $leaderboardEntryFour = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userFour->ID,
+        ]);
+
+        $userFive = User::factory()->create(['User' => 'myUser5']);
+        $leaderboardEntryFive = LeaderboardEntry::factory()->create([
+            'leaderboard_id' => $leaderboard->ID,
+            'user_id' => $userFive->ID,
+        ]);
+
+        $this->get($this->apiUrl('GetLeaderboardSubmissions', ['i' => $leaderboard->ID]))
+            ->assertSuccessful()
+            ->assertJson([
+                'Count' => 5,
+                'Total' => 5,
+                'Results' => [
```

Comment:
Using the id to check for the "Rank" value adds brittleness to the test case.

I think it would be a good idea to add at least one entry in the test case that is a tie (to get coverage that Rank is actually working correctly). In the assert, it's probably a better practice to hardcode what the rank should be for each entry.